openshift/cirras-underwriting-ui-deployment.yaml

```diff
@@ -98,6 +98,14 @@ spec:
                 configMapKeyRef:
                   name: cirras-underwriting-api-config-#{ENV}#
                   key: LOG_LEVEL
+          livenessProbe:
+            httpGet:
+              scheme: HTTP
+              path: /pub/cirras-underwriting/
+              port: #{TARGET_PORT}#
```

Comment:
I believe TARGET_PORT was renamed to TOMCAT_PORT.

Reply:
Thank you, resolved